config/routes.rb

```diff
@@ -12,4 +12,7 @@
   post '/github_event_handler', to: 'webhook#handle'
   get '/development_metrics', to: 'development_metrics#index'
   get 'tech_blog', to: 'tech_blog#index'
+  namespace :code_owners do
```

Comment:
Can we name this route something basic like `/users/:id`

It's more RESTful and also, we will use this page for multiple user metrics, so it will be kind of the user's page.